### ISSUE TITLE
Is image augmentation applied to the validation set?

### ISSUE THREAD

**Issue by Mahi-Mai** [matterport/Mask_RCNN#2057]
So, say I implemented augmentation in the manner seen here: https://github.com/matterport/Mask_RCNN/issues/1924#issuecomment-568200836

Is augmentation only applied to the training set, or is it also applied to the validation set?  Is there a way to control which dataset it's applied to, or apply a different set of augmentation to the validation set?

Thanks!

**Comment by bpmsilva** [comment]
I don't think the augmentation is applied to the validation set, because in the train function the augmentation is only passed to the `train_generator`:

```
train_generator = data_generator(train_dataset, self.config, shuffle=True,
                                 augmentation=augmentation,
                                 batch_size=self.config.BATCH_SIZE,
                                 no_augmentation_sources=no_augmentation_sources)
val_generator = data_generator(val_dataset, self.config, shuffle=True,
                                batch_size=self.config.BATCH_SIZE)
```

The code current implementation doesn't provide any control, so you'll need to change the code if you want to apply augmentation on the validation set
